app/src/components/Adoptions.js

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+
+import * as apiClient from "../apiClient";
+
+import AnimalList from "./AnimalList";
+import VolunteerButton from "./VolunteerButton";
+
+const Adoptions = () => {
+  const [animals, setAnimals] = React.useState([]);
+
+  const loadAnimals = async () => setAnimals(await apiClient.getAnimals());
+
+  React.useEffect(() => {
+    loadAnimals();
+  }, []);
+
+  console.log(animals);
```

Comment:
Try not to commit any console.log if possible.

I like using `git add -p` to go through hunk by hunk (since this a new file, it wouldn't help in this case though)